packages/makeswift-next-plugin/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @makeswift/next-plugin
 
+## 0.2.0
+
+### Minor Changes
+
+- c3041ff: Adds Next.js version checking, and transpile package support for Next.js 13.
+
```

Comment:
Can we add some more info under the release header? "Adds Next.js version checking" isn't totally clear.

Maybe we want to say something like this:

BREAKING CHANGE: The Next.js plugin will now check the version of Next.js being used and if it is less than v12.2.0 an error is thrown.

Reply:
Added